javascript/react/fundamentals/the-react-component.md

```diff
@@ -27,58 +27,77 @@ aspects:
 
 ---
 
-# The **React** component
+# The React component
 
 ---
 ## Content
 
-In **React** a `component` represents a fundamental *class* you can work with to build re-usable and independent `UI` blocks.
+In **React**, a component represents a fundamental *class* you can work with to build re-usable and independent `UI` blocks.
 
-These `component`s behave like simple functions, they can take input arguments called `props` and will return what should be *rendered*.
+These components behave like simple functions, they can take input arguments called `props` and will return what should be *rendered*.
 
-The *rendered* output needs to be specified inside a function called `render()`.
+If you want to create a simple component, you can use a JavaScript function like this:
 
-To create a component you can `extend` the `ES6`[1] class `React.Component`:
+```jsx
+// functional component
+function Enki(props) {
+  return <p>Enki aka Ea</p>;
+}
 ```
+
+At the same time, you can also `extend` the `ES6`[1] class `React.Component` to create a component. In this case, the *rendered* output needs to be specified inside a function called `render()`:
+
+```jsx
+// class component
 class Enki extends React.Component {
   render() {
     return <p>Enki aka Ea</p>;
   }
 }
 ```
 
-You can pass dynamic values to a component via `props` which will be discussed next. In addition, you can use other components when `render`ing.
+The differences between functional and class components will be discussed at the end of this workout, but for now know that React views both of the components defined above as being the same.
```

Comment:
So this is kind of true but not really.

Shall we add a footnote caveat? 

See: https://overreacted.io/how-are-function-components-different-from-classes/

Reply:
Or we could say something like:

```suggestion
The differences between functional and class components will be discussed at the end of this workout, but for now know that, when viewed externally, each way of writting a component can render the same result.
```

Reply:
We could re-write the entire insight on functional components to explain the differences and present an example similar to the one in the article. Do you think this is a good idea?